.github/workflows/code-scan-cron.yml

```diff
@@ -0,0 +1,12 @@
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  gitleaks:
+    uses: smallstep/workflows/.github/workflows/gitleaks.yml@main
+    secrets:
+      GITLEAKS_LICENSE_KEY: ${{ secrets.GITLEAKS_LICENSE_KEY }}
+
+  codeql:
+    uses: smallstep/workflows/.github/workflows/codeql-analysis.yml@main
```

Comment:
You just deleted that file? 

Also, is there any point in having all these fails in .github? Doesn't it make more sense to be directly on the root?

Reply:
So the uses is simpler, `uses: smallstep/workflows/ci.yml`

Reply:
Ah ok, this is crypto !!!!